src/Controller/AppController.php

```diff
@@ -55,18 +55,21 @@ public function initialize()
     /**
      * Before render callback.
      *
-     * @param \Cake\Event\Event $event The beforeRender event.
+     * @param  \Cake\Event\Event $event The beforeRender event.
      * @return \Cake\Http\Response|null|void
      */
     public function beforeRender(Event $event)
     {
         // Note: These defaults are just to get started quickly with development
-        // and should not be used in production. You should instead set "_serialize"
-        // in each action as required.
-        if (!array_key_exists('_serialize', $this->viewVars) &&
-            in_array($this->response->type(), ['application/json', 'application/xml'])
```

Comment:
new Cakephp released 
you can replace this by `$this->response->getType()` now

Reply:
Yeah thank you, I was waiting for the tag. I start working on this ASAP 👌